src/components/filter-accordion/filter-accordion.tsx

```diff
@@ -33,7 +33,7 @@ export const FilterAccordion: FC<IFilterAccordionProps> = ({
 
   const currentField = watch();
 
-  const getClassName = (index: number, array: any[]) => {
+  const handleGetClass = (index: number, array: any[]) => {
```

Comment:
🦆 For utilities, you can use a convention to easily identify them. For instance, if you are trying to calculate some value. You can use `get-` prefix --> `getClassName`